plugins/lookup/bitwarden.py

```diff
@@ -12,6 +12,8 @@
     requirements:
       - bw (command line utility)
       - be logged into bitwarden
+      - bitwarden vault unlocked
+      - BW_SESSION env. var. set
```

Comment:
```suggestion
      - C(BW_SESSION) environment variable set
```

Reply:
Thanks for the suggestion.  I'm not sure what it does, but I'll take your expertise over my own :smile:

Reply:
See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#format-macros-within-module-documentation for information.